src/router-view.js

```diff
@@ -37,17 +37,23 @@ const swapStrategies = new SwapStrategies();
 
 @customElement('router-view')
 @noView
-@inject(DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction)
+@inject(DOM.Element, Container, ViewSlot, Router, ViewLocator, CompositionTransaction, CompositionEngine)
 export class RouterView {
   @bindable swapOrder;
+  @bindable layout;
+  @bindable layoutView;
```

Comment:
@charlespockert Great. And what are your thoughts on using `layoutView`? `compose` has the same 3 properties, so users will be familiar with it.